src/wasm/kotlin/README.md

```diff
@@ -1,12 +1,12 @@
 # Kotlin --> WASM particles
 
 ## Build & Test
-0. Make sure the project is [setup](../../../README.md#install)!
-1. `bazel build particles/Native/Wasm:all`
-2. `npm start`
-3. Visit [localhost:8786/shells/dev-shell/?m=https://$particles/Native/Wasm/wasm.arcs](http://localhost:8786/shells/dev-shell/?m=https://$particles/Native/Wasm/wasm.arcs)
+1. [`npm install && ./tools/sigh webpack`](../../../README.md#install)
+1. `cd src/wasm/kotlin`
```

Comment:
These build rules incorrect. Gradle should no longer exist; we need to follow the bazel rules.